### ISSUE TITLE
Add option to delete a data set

### ISSUE THREAD

**Issue by cjh1** [OpenChemistry/distiller#66]

**Comment by ercius** [comment]
Here is a safe way I have been using, but we only really need the `rm` commands:
```bash
#!/bin/bash
# Delete a dataset safely from NVME drives.
# Deletes the data and log (preview) files
# inputs: base path, scan number
# example: ./delete_data.sh 5

if [[ $# -ne 1 ]]; then
    echo "No scan selected" >&2
    exit 2
fi

echo Number of files to delete:
ls /mnt/nvmedata*/data_scan0*0$1_module*_dst*_file*.data | wc -l

echo note: 1kx1k = 288 files,  512x512 and smaller = 72 files
read -p "Delete this data (y/n)? " -r

echo 
if [[ $REPLY =~ ^[Yy]$ ]]
then
    echo Deleting scan $1
    rm /mnt/nvmedata*/data_scan0*0$1_module*_dst*_file*.data
    rm /mnt/nvmedata*/log_scan0*0$1_module*_dst*_file*.data
fi
echo Done
```

**Comment by ercius** [comment]
I found a bad bug in the previous code. This is a better way to delete the data sets:

```bash
#!/bin/bash
# Delete a dataset safely from NVME drives.
# Deletes the data and log (preview) files
# inputs: base path, scan number
# example: ./delete_data.sh 5

if [[ $# -ne 1 ]]; then
    echo "No scan selected" >&2
    exit 2
fi

echo Number of files to delete:
ls /mnt/nvmedata*/data_scan`printf %010d $1`_module*_dst*_file*.data | wc -l

echo note: 1kx1k = 288 files,  512x512 and smaller = 72 files
read -p "Delete this data (y/n)? " -r

echo 
if [[ $REPLY =~ ^[Yy]$ ]]
then
    echo Deleting scan $1
    rm /mnt/nvmedata*/data_scan`printf %010d $1`_module*_dst*_file*.data
    rm /mnt/nvmedata*/log_scan`printf %010d $1`_to*_module*_dst*_file*.data
fi
echo Done
```

**Comment by cjh1** [comment]
Implemented in #122
